.github/workflows/deployment.yml

```diff
@@ -28,3 +29,14 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         BRANCH: gh-pages
         FOLDER: documentation
+    - name: Publish to npm
```

Comment:
You only want to execute that usually when you create a release.

Or at least only on master if it won't error when the version already exists.